main/resource-access.bs

```diff
@@ -279,6 +279,25 @@ Issue:
 Pertaining to events and loss of control mitigation:
 https://github.com/solid/specification/issues/41#issuecomment-534679278
 
+### Representations ### {#representations}
+
+When a server creates a resource on HTTP `PUT`, `POST` or `PATCH` requests
+such that the request's representation data encodes an *RDF document*
+[[!RDF11-CONCEPTS]] (as determined by the `Content-Type` header), the server
+MUST accept `GET` requests on this resource when the value of the `Accept`
+header requests a representation in `text/turtle` or `application/ld+json`
+[[!Turtle]] [[!JSON-LD11]].
+[[Source](https://github.com/solid/specification/issues/45)]
+[[Source](https://github.com/solid/specification/issues/69)]
+[[Source](https://github.com/solid/specification/issues/109)]
+[[Source](https://github.com/solid/specification/issues/195)]
+
+When a `PUT`, `POST`, `PATCH` or `DELETE` method request targets a resource's
+representation URL, the server MUST respond with a `307` or `308` status code
```

Comment:
The redirect requirement makes sense in the case where the resource URL (e.g. `http://example.com/foo`) is different from the representation URL (e.g. `http://example.com/foo.jsonld`), but what if all representations are fetched from the same resource URL (`/foo`)? Should this sentence include something like "if the representation URL is different than the resource URL"? Or perhaps the term "representation URL" should be more rigorously defined?

Reply:
I agree that'd be good to distinguish. Initial update here: https://github.com/solid/specification/pull/196/commits/e92d8bce5caaad42fcaad465ba326e44fc1357fc . Will this do for now? I like the idea of further clarifying how we are using "resource" and "representation" but we shouldn't define per se - perhaps just remind RFC's notions.

Reply:
@csarven thanks, that update works for me